src/main/java/com/google/api/codegen/discovery/transformer/go/GoSampleMethodToViewTransformer.java

```diff
@@ -96,7 +96,10 @@ private SampleView createSampleView(SampleTransformerContext context) {
       fieldVarNames.add(requestBodyVarName);
     }
 
-    if (methodInfo.isPageStreaming()) {
+    // The Go client only considers methods whose verb is "GET" to be page
```

Comment:
At the moment, yes
https://github.com/google/google-api-go-client/blob/master/google-api-go-generator/gen.go#L1398